tests/shared.py

```diff
@@ -186,10 +186,14 @@ def run(self, *command):
 
 
 class GitRepo(Repo):
-    def __init__(self, content_dir):
+    def __init__(self, content_dir, init_default_branch=None):
         super().__init__(content_dir)
 
-        self.run('git', 'init')
+        branch_args = []
+        if init_default_branch is not None:
+            branch_args = ['--initial-branch', init_default_branch]
+        # Note the * character. This has no effect if the list is empty.
+        self.run('git', 'init', *branch_args)
```

Comment:
Now that I'm testing this, I'm reminded that our `test.py` script has been spitting out a ton of warnings about how the "default branch is subject to change". Maybe the right thing to do would be to just always pass in `--initial-branch`, to suppress these warnings? So maybe the default value could be `"master"` for now, instead of `None`?

Reply:
Why not?